viewer/src/public/migration/CognitePointCloudModel.ts

```diff
@@ -11,12 +11,18 @@ import { PotreeGroupWrapper } from '@/datamodels/pointcloud/PotreeGroupWrapper';
 import { PotreePointColorType } from '@/datamodels/pointcloud/types';
 import { Matrix4 } from 'three';
 
+/**
+ * Documentation for the Cognite3DModel class
```

Comment:
@j-bjorne @larsmoa @christjt I'm not really sure what to write as a general overview for this class and for the cad model class as well. Would be happy to get some suggestions on it.